airflow/jobs/scheduler_job.py

```diff
@@ -65,8 +65,8 @@ def __init__(self, file_path, pickle_dags, dag_id_white_list):
         :type file_path: unicode
         :param pickle_dags: whether to serialize the DAG objects to the DB
         :type pickle_dags: bool
-        :param dag_id_whitelist: If specified, only look at these DAG ID's
-        :type dag_id_whitelist: list[unicode]
+        :param dag_id_white_list: If specified, only look at these DAG ID's
+        :type dag_id_white_list: list[unicode]
```

Comment:
Can you move the parameter description to the class description?  It is not correct to have a separate class description and constructor. Sphinx does not support this properly.

Reply:
Done